packages/grid/x-data-grid/src/tests/sorting.DataGrid.test.tsx

```diff
@@ -91,6 +91,67 @@ describe('<DataGrid /> - Sorting', () => {
     expect(getColumnValues()).to.deep.equal(['Adidas', 'Puma', 'Nike']);
   });
 
+  it('should only allow ascending sorting using sortingOrder', () => {
```

Comment:
Thanks for adding tests for the component-wide `sortingOrder`. 💪

Reply:
No problem - I did notice some funky (existing) behaviour when `null` is used in `sortingOrder` in somewhere other than the last position, e.g. `['asc', null, 'desc']` will always cycle between sorting by ascending and the initial order, never descending - I wonder if this should be noted in the docs or even throw an error at runtime.

Reply:
I guess this prop should only accept `null` at the edges, never in the middle. It might be confusing to go from ascending, no sorting applied, and descending. Let's wait for someone to open an issue with an use case that requires `null` in this position.